features/batch/step_definitions.rb

```diff
@@ -0,0 +1,79 @@
+# Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not
+# use this file except in compliance with the License. A copy of the License is
+# located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on
+# an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions
+# and limitations under the License.
+
+And(/^a (Parent|Child) model with TableConfig of:$/) do |model, code_block|
+  case model
+  when 'Parent'
+    ParentTableModel = @parent
+  when 'Child'
+    ChildTableModel = @model
+  else
+    raise 'Model must be either a Parent or Child'
+  end
+
+  @table_config = eval(code_block)
+end
+
+When(/^we make a batch write call with following Parent and Child model items:$/) do |string|
+  item_data = JSON.parse(string, symbolize_names: true)
+
+  Aws::Record::Batch.write do |db|
+    item_data.each do |item|
+      case item[:model]
+      when 'Parent'
+        formatted_item = remove_model_key(item)
+        item_instance = @parent.new(formatted_item)
+      when 'Child'
+        formatted_item = remove_model_key(item)
```

Comment:
Could maybe combine a few lines here and do just:
```
db.put(@parent.new(remove_model_key(item)))
```

Reply:
Completed! So much shorter, nice.